e2e/_suites/ingest-manager/fleet.go

```diff
@@ -83,6 +85,25 @@ func (fts *FleetTestSuite) anAgentIsDeployedToFleet() error {
 	return err
 }
 
+// downloadAgentBinary it downloads the binary and stores the location of the downloaded file
+// into the Fleet struct, to be used else where
+func (fts *FleetTestSuite) downloadAgentBinary() error {
+	artifact := "elastic-agent"
+	version := "8.0.0-SNAPSHOT"
```

Comment:
This version should be configurable, to support running the test suite against multiple versions of the agent.

I could think of an env var with the URL. If it is present in the execution, simply use it.